Notifier-service/src/models/conference.ts

```diff
@@ -1,65 +1,61 @@
+export class ConferenceModelI extends ConferenceModel {
     modelName = "conference"
-    private model:Promise<mongoose.Model<ConferenceDocument,{}>>;
-    private connection:Promise<Connection>;
+    private model: Promise<mongoose.Model<ConferenceDocument, {}>>;
+    private connection: Promise<Connection>;
     private logger = new Logger(this.constructor.name).getLogger();
-    constructor(database:Database){
+    constructor(database: Database) {
         super(database)
-        
-        // this.connection = connection
-        // this.model = this.connection.model<ConferenceDocument>( this.modelName, ConferenceSchema);
-
         this.connection = database.getConnection()
-        .then( (connection:Connection) => {
-            return Promise.resolve(connection);
-        } )
-        .catch((error) => {
-            let errstring = "Failed at getting connection :"+error;
-            this.logger.error(errstring);
-            return Promise.reject(error);
-        })
+            .then((connection: Connection) => {
+                return Promise.resolve(connection);
+            })
+            .catch((error) => {
+                let errstring = "Failed at getting connection :" + error;
+                this.logger.error(errstring);
+                return Promise.reject(error);
+            })
 
         this.model = this.connection
-        .then( (connection: Connection) =>{
-            let model = connection.model<ConferenceDocument>( this.modelName, ConferenceSchema);
-            return Promise.resolve(model);
-        } )
-        .catch((error) => {
-            let errstring = "Failed at getting connection for model"+error;
-            this.logger.error(errstring);
-            return Promise.reject(error);
-        })
+            .then((connection: Connection) => {
+                let model = connection.model<ConferenceDocument>(this.modelName, ConferenceSchema);
+                return Promise.resolve(model);
+            })
+            .catch((error) => {
+                let errstring = "Failed at getting connection for model" + error;
+                this.logger.error(errstring);
+                return Promise.reject(error);
+            })
 
     }
-    
-    private async  makeQuery<T>(callback:(model:mongoose.Model<ConferenceDocument,{}>) => Promise<T>):Promise<T> {
+
+    private async  makeQuery<T>(callback: (model: mongoose.Model<ConferenceDocument, {}>) => Promise<T>): Promise<T> {
         return this.model
-        .then(callback)
-        .catch(error => {
-            this.logger.debug("Failed at getOne: model not initialised error:"+error);
-            this.logger.error("Failed at getOne : model must have failed to initialize :"+error);
-            return Promise.reject(new Error("model failed to be initialised"));
-        });                
+            .then(callback)
+            .catch(error => {
+                this.logger.debug("Failed at getOne: model not initialised error:" + error);
```

Comment:
* Change the error logger to reflect which function has failed. 
* "Failed at getOne"  should change to "Failed at --callbackname--"